libs/common-services/src/main/java/uk/gov/di/ipv/core/library/domain/CriJourneyRequest.java

```diff
@@ -1,19 +1,21 @@
 package uk.gov.di.ipv.core.library.domain;
 
-import lombok.Builder;
+import lombok.AllArgsConstructor;
 import lombok.Data;
 import lombok.EqualsAndHashCode;
 import lombok.NoArgsConstructor;
+import lombok.experimental.SuperBuilder;
 import uk.gov.di.ipv.core.library.annotations.ExcludeFromGeneratedCoverageReport;
 
 @ExcludeFromGeneratedCoverageReport
 @EqualsAndHashCode(callSuper = true)
+@AllArgsConstructor
```

Comment:
Do you need this with the explicit constructor below?

Reply:
I did need it with the `@Builder` annotation (`Lombok @Builder needs a proper constructor for this class `), but it seems the `@SuperBuilder` can take either constructor, so I will go with removing the explicit constructor